src/lib/web-component/filter-option/filter-option-element.js

```diff
@@ -95,8 +95,6 @@ export class FilterOption extends HTMLElement {
       }
 
       this.dispatchFilterChangeEvent();
-    } else {
-      throw new Error(`wrong event type - ${event.type}`);
     }
```

Comment:
@atiqueahmedziad what are the reasons behind this removal? was this actually triggered in any of the test cases?

Reply:
No, this hasn't been triggered by any of the test cases.
Currently, we are not catching this error anywhere (we also didn't set any UI to show this error in the `filter-option` web component) and this error is very less likely to get produced.